src/protocol/ipa_prf/boolean_ops/addition_sequential.rs

```diff
@@ -34,7 +33,8 @@ where
     XS::Element: Field,
 {
     let mut carry = AdditiveShare::<XS::Element>::ZERO;
-    addition_circuit(ctx, record_id, x, y, &mut carry).await
+    let sum = addition_circuit(ctx, record_id, x, y, &mut carry).await?;
+    Ok((carry, sum))
```

Comment:
The ordering of these two threw me here.  Most other addition functions return (sum, carry), see i8::carrying_add or any of the other standard library functions.